style-guide/src/layouts/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Link from 'gatsby-link';
+
+import styles from '../styles/index.module.css';
+
+const NavLink = props => (
+	<div className={styles['nav-link']}>
+		<Link to={props.to} activeClassName={styles['link-active']}>
+			{props.children}
+		</Link>
+	</div>
+);
+
+export default ({ children }) => (
+	<div className={styles['two-column']}>
+		<div className={styles['nav']}>
+			<Link className={styles['link-secondary']} to="/">
+				<h1>OTKit Style Guide</h1>
+			</Link>
+			<NavLink to="/otkit-desktop-typography/">Typography</NavLink>
+			<NavLink to="/otkit-colors/">Colors</NavLink>
+		</div>
+		<div className={styles['main']}>{children()}</div>
```

Comment:
I think it's more idiomatic to write `{children}`. You don't execute it. Actually, what does executing it even do? I know that `children` is not really an array, but is it a function...?

Reply:
@lastquestion This is a feature of Gatsby. See https://www.gatsbyjs.org/tutorial/part-three/#our-first-layout-component

Reply:
Wow. Thanks, TIL. I __did not know this__ for sure. Interesting design choice...